examples/java/brave/src/main/java/com/tracing/TraceExample.java

```diff
@@ -182,6 +184,10 @@ public void handle(HttpExchange t) throws IOException {
 
 				String response = "This is path A";
 				t.sendResponseHeaders(200, response.length());
+				span.tag("http.url", t.getRequestURI().toString());
```

Comment:
in real life we don't do things like this manually. usually you want all http tagging policy the same. This is why we have adapters. With adapters you can decouple tagging policy from the instrumentation point https://github.com/openzipkin/brave/tree/master/instrumentation/http#http-server

Reply:
Should adapters be part of joint API then?